grakn-test/test-snb/src/main/java/ai/grakn/GraknQueryHandlers.java

```diff
@@ -156,14 +158,14 @@ public void executeOperation(LdbcQuery8 ldbcQuery8, GraknDbConnectionState dbCon
             GraknSession session = dbConnectionState.session();
             try (GraknTx graknTx = session.open(GraknTxType.READ)) {
                 // for speed the query is again split into the ordering and limit phase
-                Var $reply = var("aReply");
-                Var $responder = var("responder");
-                Var $responderId = var("responderId");
+                Var reply = var("aReply");
+                Var responder = var("responder");
+                Var responderId = var("responderId");
                 Match orderQuery = match(
                         $person.has(PERSON_ID, ldbcQuery8.personId()),
                         var().rel($person).rel($message).isa(HAS_CREATOR),
-                        var().rel($message).rel(REPLY, $reply).isa(REPLY_OF),
-                        $reply.has(CREATION_DATE, $date).has(MESSAGE_ID, $messageId)
+                        var().rel($message).rel(REPLY, reply).isa(REPLY_OF),
+                        reply.has(CREATION_DATE, $date).has(MESSAGE_ID, $messageId)
```

Comment:
We only changed some of the dollars. Should probably be all or nothing.

Reply:
Working on the rest in a separate PR. I am going according to the order provided by sonar